third_party/CMakeLists.txt

```diff
@@ -70,7 +70,7 @@ else()
     aws_sdk
     SOURCE_DIR ${aws_cpp_sdk_SOURCE_DIR}
     INSTALL_DIR ${aws_cpp_sdk_INSTALL}
-    CMAKE_ARGS "-DBUILD_SHARED_LIBS=OFF" "-DBUILD_ONLY=transfer;s3" "-DENABLE_TESTING=OFF" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=${aws_cpp_sdk_INSTALL}"
+    CMAKE_ARGS "-DBUILD_SHARED_LIBS=OFF" "-DBUILD_ONLY=transfer;s3" "-DENABLE_TESTING=OFF" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=${aws_cpp_sdk_INSTALL}" "-DCMAKE_INSTALL_LIBDIR=lib"
```

Comment:
`CMAKE_INSTALL_LIBDIR` is needed because the lib directory for AWSSDK can vary between `lib` and `lib64`